### ISSUE TITLE
Persistent WI Filters

### ISSUE THREAD

**Issue by brigadecs** [forcedotcom/devops-center-roadmap#114]
Currently, when applying filters to WIs, they are ephemeral and revert back to all work items resulting in a long list of all WIs, even if they're Nevered or Completed. There should be means to have filters applied permanently and options to have multiple filter views; Similar to List Views.

**Comment by oleg-mastriukov** [comment]
Agree, this is so annoying. In most cases, I just want to open the last WI, so I have to change the sorting order twice (ASC -> DESC) every time I open the WI list.

**Comment by SE-Sleek** [comment]
Agree this would be a major QOL improvement. I do believe this is just a list view, I think? So it seems that allowing list-view like functionality like creation of ad hoc filtered views, and allowing users to pin their preferred view, would be possible. I would LOVE This!

**Comment by ajmujaa** [comment]
i agree to this one as well. It'll make the life much easier. If we have the same list view functioanlity as it's int he Salesforce, it would be really great.

**Comment by fmannreply** [comment]
This is becoming increasingly important as the list of completed items grows. It seams like a low-hanging fruit.

**Comment by InnovAgeAlon** [comment]
I agree with what everyone is saying - one of the most obvious and universally useful changes that could be made. Why this is only in the "Considering" category is beyond me... I expect literally every person who's ever user DevOps Center for more than a few days wants this feature. I hope this moves up the Roadmap quickly, and thank you to anyone who makes that happen!

**Comment by mathieu-lanefour** [comment]
Agreed, this really hinders the tool the default should at least be DESC order for work items. But persistent filters so we can remove the nevers and completed work items is really important too
